tutorials/ngsi_v2/fiware_env/docker-compose.yml

```diff
@@ -0,0 +1,96 @@
+
+volumes:
+    mongodb: ~
+    crate-db: ~
+
+services:
+    iot-agent:
+        image: fiware/iotagent-json:1.26.0
+        hostname: iot-agent
+        container_name: fiware-iot-agent
+        depends_on:
+            - mongodb
+            - mosquitto
+        expose:
+            - "4041"
+        ports:
+            - "4041:4041"
+        environment:
+            - "IOTA_CB_HOST=orion"
+            - "IOTA_CB_PORT=1026"
+            - "IOTA_NORTH_PORT=4041"
+            - "IOTA_REGISTRY_TYPE=mongodb"
+            - "IOTA_MONGO_HOST=mongodb"
+            - "IOTA_MONGO_PORT=27017"
+            - "IOTA_MONGO_DB=iotagent-json"
+            - "IOTA_PROVIDER_URL=http://iot-agent:4041"
+            - "IOTA_MQTT_HOST=mosquitto"
+            - "IOTA_MQTT_PORT=1883"
+            - "IOTA_DEFAULT_RESOURCE="
+            - "IOTA_DEFAULT_TRANSPORT=MQTT"
+            - "IOTA_LOG_LEVEL=DEBUG"
+            - "IOTA_TIMESTAMP=true"
+            - "IOTA_CB_NGSI_VERSION=v2"
+            - "IOTA_AUTOCAST=true"
+
+    mongodb:
+        image: mongo:4.4
+        hostname: mongodb
+        container_name: db-mongo
+        ports:
+            - "27017:27017"
+        volumes:
+            - mongodb:/data
+
+    orion:
+        image: fiware/orion:3.7.0
+        hostname: orion
+        container_name: fiware-orion
+        depends_on:
+            - mongodb
+        expose:
+            - "1026"
+        ports:
+            - "1026:1026"
+        command: 
+          -dbhost mongodb
+          -logLevel DEBUG
+          -logForHumans
+
+    mosquitto:
+        image: eclipse-mosquitto:2.0.18
+        hostname: mosquitto
+        container_name: mosquitto
+        expose:
+          - "1883"
+          - "9001"
```

Comment:
Why is the port 9001 exposed?

Reply:
Port 9001 is the standard port for HTTP/Websocket communications